python/src/StandardSpaceCrossEntropyImportanceSampling_doc.i.in

```diff
@@ -0,0 +1,98 @@
+    Intermediate quantile level. The default number can be tweaked with the `CrossEntropyImportanceSampling-DefaultQuantileLevel` key from :class:`~openturns.ResourceMap`.
+    
+Notes
+-----
+The following explanations are given for a failure event defined as :math:`g(\mathbf{X})<T` with :math:`\mathbf{X}` 
+a random vector following a joint PDF :math:`f_\mathbf{X}`, :math:`T` a threshold and :math:`g` a limit state function,
+without loss of generality.
+
+The Importance Sampling (IS) probability estimate :math:`\widehat{P}^\text{IS}` is given by:
+
+.. math::
+
+    \widehat{P}^\text{IS}=\frac{1}{N} \sum_{i=1}^{N} {\mathbf{1}}_{g(\mathbf{x}_i)<T} \frac{f_\mathbf{X}(\mathbf{x}_i)}{h(\mathbf{x}_i)},
+
+with :math:`h` the auxiliary PDF of Importance Sampling, 
+:math:`N` the number of independent samples generated with :math:`h` and :math:`{\mathbf{1}}_{g(\mathbf{x}_i)<T}` the 
+indicator function of the failure domain. 
+
+The optimal density :math:`h_{opt}` minimizing the variance of the estimator :math:`h_{opt}` is defined as:
+
+.. math::
+
+    h_{opt}=\frac{{\mathbf{1}}_{g(\mathbf{x})<T}f_\mathbf{X}}{P},
+
+with :math:`P` the failure probability which is inaccessible in practice since this probability is the quantity of interest and unknown.
+
+The Standard Space Cross-Entropy Importance Sampling [rubinstein2017]_ uses a parametric auxiliary distribution :math:`h_\mathbf{\lambda}` in order to optimize its parameters to compute the probability of interest with accuracy. 
+This process is performed in the standard space using a Normal distribution as the auxiliary distribution. Indeed, in the standard space, the optimal value of the mean and standard deviations are given analytically, see [morio2015]_ for details. 
+
+We note  :math:`G(\mathbf{u})` the limit state function in standard space and  :math:`g(\mathbf{x})` its counterpart in the physical space. :math:`f_\mathbf{U}` stands for the initial distribution in standard space, i.e. the Normal law. The algorithm is as follows:
+
+
+1. :math:`k=1`, set the quantile level :math:`\rho \in [0,1]` and :math:`h_0 = f_{\mathbf{U}}`
+
+2. Generate the population :math:`\mathbf{u}_1^{(k)},...,\mathbf{u}_N^{(k)}` according to the PDF :math:`h_{k-1}`, apply the 
+   function :math:`G` in order to have :math:`y_1^{(k)}=G(\mathbf{u}_1^{(k)}),...,y_N^{(k)} = G(\mathbf{u}_N^{(k)})`
+
+3. Compute the   :math:`\rho`-quantile :math:`q_k=\max(T,y^{(k)}_{\left \lfloor \rho N \right\rfloor})` using the floor of :math:`\rho N`.
+
+4. If :math:`q_k>T`, go to Step 7
+
+4. Update the means and standard deviations of the auxiliary distribution:
+
+.. math::
+
+       \mu_{k}= \frac{\frac{1}{N} \sum_{i=1}^{N}{\mathbf{1}}_{G(\mathbf{u}_i^{(k)}) \leq q_k} \frac{f_\mathbf{U}(\mathbf{u}_i^{(k)})}{h_{\mathbf{\lambda}_{k-1}}(\mathbf{u}_i^{(k)})} U_{ik}}{\frac{1}{N} \sum_{i=1}^{N}{\mathbf{1}}_{G(\mathbf{u}_i^{(k)}) \leq q_k} \frac{f_\mathbf{U}(\mathbf{u}_i^{(k)})}{h_{\mathbf{\lambda}_{k-1}}(\mathbf{u}_i^{(k)})} }
```

Comment:
What is U_{ik}? Is it the same as \mathbf{u}_i^{(k)}) ?

Reply:
Typo fixed.